server/log_operation_manager.go

```diff
@@ -19,20 +19,39 @@ import (
 	"fmt"
 	"math/rand"
 	"reflect"
+	"strconv"
 	"sync"
 	"time"
 
 	"github.com/golang/glog"
 	"github.com/google/trillian/extension"
+	"github.com/google/trillian/monitoring"
 	"github.com/google/trillian/util"
 )
 
 const (
 	minPreElectionPause    = 10 * time.Millisecond
 	minMasterCheckInterval = 50 * time.Millisecond
 	minMasterHoldInterval  = 10 * time.Second
+	logIDLabel             = "logid"
 )
 
+var (
+	once         sync.Once
+	knownLogs    monitoring.Gauge
+	resignations monitoring.Counter
+	isMaster     monitoring.Gauge
+)
+
+func createMetrics(mf monitoring.MetricFactory) {
+	if mf == nil {
+		mf = monitoring.InertMetricFactory{}
+	}
+	knownLogs = mf.NewGauge("known_logs", "Set to 1 for known logs", logIDLabel)
```

Comment:
Can you just clarify what known_logs is? Looks like the logs this signer is master for?